index.js

```diff
@@ -90,3 +91,4 @@ module.exports.upgrade = makeDispatcher(api.upgrade)
 module.exports.MockClient = MockClient
 module.exports.MockPool = MockPool
 module.exports.MockAgent = MockAgent
+module.exports.mockErrors = mockErrors
```

Comment:
I've exposed the `mockErrors` as a separate object to `errors` so they are not confused, happy to refactor if you would prefer them to be all together :)